packages/opentelemetry-exporter-collector/src/platform/node/CollectorExporterNodeBase.ts

```diff
@@ -33,13 +33,17 @@ export abstract class CollectorExporterNodeBase<
 > {
   DEFAULT_HEADERS: Record<string, string> = {};
   headers: Record<string, string>;
+  keepAlive: boolean = false;
```

Comment:
This is a question for everyone, but is there any reason why we shouldn't default this to `true`?

Reply:
Yes. I also had same doubt here. I am keeping the default here same as the default in Node. Let's others also review and input their comments on this.

Reply:
I agree it make senses to default to true.